scripts/feature_extractor_static.py

```diff
@@ -2,13 +2,32 @@
 
 
 class StaticEmbeddingExtractor:
+    """
+    this class includes methods to extract pretrained static word embeddings
+    the pretrained embeddings stem from finalfusion
+    :param: path_to_fifu: the path to the pretrained embeddings stored with .fifu format
+    """
 
     def __init__(self, path_to_fifu):
-        self._embeds = finalfusion.Embedding(path_to_fifu, mmap=True)
+        self._embeds = finalfusion.Embeddings(path_to_fifu, mmap=True)
```

Comment:
might be useful to be able to load fastext and w2vec as well, so that we don't have to have them in fifu format (as the python binding allows for loading w2vec and .bin format). you can check the ending of the file specified by the user and call the corresponding loading method. plus assert that the ending is either fifu, bin or w2v